app/data_model/answer_store.py

```diff
@@ -0,0 +1,192 @@
+import re
+
+from collections import OrderedDict
+
+
+class Answer(object):
+    def __init__(self, answer_id, block_id, value, answer_instance=None):
+        answer_instance = answer_instance or 0
+
+        self.answer_id = answer_id
+        self.block_id = block_id
+        self.answer_instance = answer_instance
+        self.value = value
+
+    def equals(self, answer):
```

Comment:
If you do this, be aware of adding a `__hash__` function (https://docs.python.org/3.6/reference/datamodel.html#object.__hash__) and also that you don't need to define `__ne__`: "By default, __ne__() delegates to __eq__() and inverts the result unless it is NotImplemented."

You may also want some kind of test harness that checks Answer's `__dict__` property and tries changing each value in turn to ensure the operator works; past experience shows it can slip through the net to update the operator when adding new properties to a class which can lead to strange/irritate behaviour (although the same applies even with the current implementation)